client/scss/generic/_normalize.scss

```diff
@@ -284,245 +115,173 @@ sup {
- *    `input` and others.
- * 4. Remove inner spacing in IE 7 without affecting normal text inputs.
- *    Known issue: inner spacing remains in IE 6.
- */
-
-button,
-html input[type="button"], /* 1 */
-input[type="reset"],
-input[type="submit"] {
-  -webkit-appearance: button; /* 2 */
-  cursor: pointer; /* 3 */
-  *overflow: visible; /* 4 */
-}
 
 /**
- * 1. Remove excess padding in IE 8/9.
- * 2. Remove excess padding in IE 7.
- *    Known issue: excess padding remains in IE 6.
+ * Correct the cursor style of increment and decrement buttons in Chrome.
  */
 
-input[type='checkbox'],
-input[type='radio'] {
-  padding: 0; /* 1 */
-  *height: 13px; /* 2 */
-  *width: 13px; /* 2 */
+[type="number"]::-webkit-inner-spin-button,
+[type="number"]::-webkit-outer-spin-button {
+  height: auto;
 }
 
 /**
- * 1. Address `appearance` set to `searchfield` in Safari 5 and Chrome.
- * 2. Address `box-sizing` set to `border-box` in Safari 5 and Chrome
- *    (include `-moz` to future-proof).
+ * 1. Correct the odd appearance in Chrome and Safari.
+ * 2. Correct the outline style in Safari.
  */
 
-input[type='search'] {
+[type="search"] {
   -webkit-appearance: textfield; /* 1 */
-  -moz-box-sizing: content-box;
-  -webkit-box-sizing: content-box; /* 2 */
-  box-sizing: content-box;
+  outline-offset: -2px; /* 2 */
 }
 
 /**
- * Remove inner padding and search cancel button in Safari 5 and Chrome
- * on OS X.
+ * Remove the inner padding in Chrome and Safari on macOS.
  */
 
-input[type='search']::-webkit-search-cancel-button,
-input[type='search']::-webkit-search-decoration {
+[type="search"]::-webkit-search-decoration {
   -webkit-appearance: none;
 }
 
 /**
- * Special styles to counteract Firefox's completely unwarranted assumptions
- * about button styles
+ * 1. Correct the inability to style clickable types in iOS and Safari.
+ * 2. Change font properties to `inherit` in Safari.
  */
 
-input[type='submit'],
-input[type='reset'],
-input[type='button'],
-button {
-  padding: 0 1em;
+::-webkit-file-upload-button {
+  -webkit-appearance: button; /* 1 */
+  font: inherit; /* 2 */
 }
 
-/**
- * Remove inner padding and border in Firefox 3+.
- */
-
-button::-moz-focus-inner,
-input::-moz-focus-inner {
-  border: 0;
-  padding: 0;
-}
+/* Interactive
+   ========================================================================== */
 
-/**
- * 1. Remove default vertical scrollbar in IE 6/7/8/9.
- * 2. Improve readability and alignment in all browsers.
+/*
+ * Add the correct display in Edge, IE 10+, and Firefox.
  */
 
-textarea {
-  overflow: auto; /* 1 */
-  vertical-align: top; /* 2 */
+details {
+  display: block;
 }
 
-/* ==========================================================================
-   Tables
-   ========================================================================== */
-
-/**
- * Remove most spacing between table cells.
+/*
+ * Add the correct display in all browsers.
  */
 
-table {
-  border-collapse: collapse;
-  border-spacing: 0;
```

Comment:
Removing this causes a lot of our tables to lose their border, or to get bigger than intended. See for example http://localhost:8000/admin/snippets/.